sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -210,6 +201,38 @@ def __init__(
         database_account = self._global_endpoint_manager._GetDatabaseAccount(**kwargs)
         self._global_endpoint_manager.force_refresh(database_account)
 
+        # Use database_account if no consistency passed in to verify consistency level to be used
+        self._set_client_consistency_level(database_account, consistency_level)
+
+    def _set_client_consistency_level(
+            self,
+            database_account: ClassType,
+            consistency_level: Optional[str],
+    ) -> None:
+        """Checks if consistency level param was passed in by user and sets it to that value or to the account default.
+
+        :param database_account: The database account to be used to check consistency levels
+        :type database_account: ~azure.cosmos.documents.DatabaseAccount
+        :param consistency_level: The consistency level passed in by the user
+        :type consistency_level: Optional[str]
+        :rtype: None
+        """
+        if consistency_level is None:
```

Comment:
Currently there is no verification for Consistency level, shouldn't we add some validation to this. Otherwise this may result in unexpected behavior. 
If python has enums, it is best to use them here, and then use enums in internal implementation. Please see java code for more details.

Reply:
what do you mean by verification? the backend will throw an error if the user attempts to pass in an invalid consistency